mesonbuild/modules/python.py

```diff
@@ -410,6 +416,19 @@ def _get_path(self, state: T.Optional['ModuleState'], key: str) -> None:
         value = state.get_option(f'{key}dir', module='python')
         if value:
             return value
+
+        install_env = state.get_option('install_env', module='python')
+        if install_env == 'auto':
+            install_env = 'venv' if self.info['is_venv'] else 'system'
+
+        if install_env == 'system':
+            rel_path = os.path.join(self.info['variables']['prefix'], rel_path)
```

Comment:
Do I understand correctly that in the `system` and `venv` cases, `rel_path` is actually an absolute path? Otherwise it's going to prepend prefix, right? Maybe change the variable name to reflect that.

Reply:
It is still treated like a relative path, as its purpose is basically to be joined with `get_option('prefix')` and assigned as abs_path further down.


What do you recommend instead?

`can_be_rel_path`?
`ans`?